app/controllers/v0/profile/ch33_bank_accounts_controller.rb

```diff
@@ -11,6 +11,19 @@ def index
         render_find_ch33_dd_eft
       end
 
+      def payload_session
```

Comment:
Also, as a FYI, generally it's an anti-pattern in Rails to define a public method in a controller that's not tied up to a route.